### PR TITLE
add Aug 2017 uPortal Slack summary

### PR DESCRIPTION
**Pull request by apetro**

### CODE REVIEW

File: _posts/uportal/2017-09-15-August-2017-uportal-slack-summary.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
++ You can't access it anonymously.
++ It's not Google search indexed. 
++ Older messages aren't available even if you log in.
+
+Summarizing the conversations here on `apereo.github.io` in this anonymously, publicly accessible, and Google-indexable context somewhat mitigates these problems. But not necessarily other problems that make [email list communications preferable in open source projects][].
+
+Arguably, all of the conversations held in the `#uportal` Slack channel could have been held via email on `uportal-dev@` or `uportal-user@` email lists additionally or instead. Some relevant email list threads are linked above.
+
+For myself, I'm convinced that we just shouldn't use Slack in the Apereo uPortal project. Use the email lists. Possibly look to implementing Discourse for better discussion forums / "email lists".
```
Comment (ChristianMurphy) [inline _posts/uportal/2017-09-15-August-2017-uportal-slack-summary.md]: :+1: for discourse

**Comment by asgrant** [comment]
@apetro I'm not sure if it is needed to make a case against Slack, it appears traffic and activity is doing that alone. My two biggest issues with Slack is the trouble it takes to get a user onto the channel in the first place and the non-transparency (i.e. not discoverable to people outside of Slack). Thank you for summarizing this and solving my second beef with Slack.

**Comment by apetro** [comment]
Slack isn't our biggest challenge by any means, but I figure it's worth pushing against it until it goes away. A barely implemented Slack for uPortal is problematic in that someone might try to engage there, get a lukewarm experience, and draw conclusions about the uPortal community from that experience, rather than realizing that that's not where it's at.

**Comment by asgrant** [comment]
I definitely see that point... I had the same opinion of some of our other public efforts as well. My thought is what can we get out of this platform that we can't get anywhere else easily.

We have Slack at OU, although it is integrated with GitLab, Trello and others that where I see the value, also there is a low barrier of entry to get into our system. I don't think that vision of how we use Slack at OU is applicable to Apereo #uportal. If we don't do integration with external systems, it is just a chat platform at that point, which is easily done elsewhere at more public and easier to join alternatives (i.e. mailing list, email, Twitter).

**Pull request merged**